tutor/resources/styles/components/modals.scss

```diff
@@ -24,7 +27,7 @@
     text-align: left;
     border-top: 0;
     button {
-      height: 50px;
+      height: 40px;
     }
```

Comment:
On my earlier changes to the modal, I'd imported the pattern library buttons.  If we're changing this should we do that?  https://github.com/openstax/tutor-js/commit/85861e418e63f70c4b2486ed0d53f7e78c3242c9#diff-6b525495a136dda2344b10d715890f04

Reply:
yes i will do htat